src/transformers/training_args.py

```diff
@@ -639,6 +639,9 @@ class TrainingArguments:
         include_inputs_for_metrics (`bool`, *optional*, defaults to `False`):
             Whether or not the inputs will be passed to the `compute_metrics` function. This is intended for metrics
             that need inputs, predictions and references for scoring calculation in Metric class.
+        eval_do_concat_batches (`bool`, *optional*, defaults to `True`):
+            If set to `False`, inputs/losses/labels/predictions are stored as lists, with each batch kept separate.
+            If set to `True`, tensors in these nested objects are recursively concatenated across batches.
```

Comment:
```suggestion
            Whether to recursively concat inputs/losses/labels/predictions across batches. If `False`,
            will instead store them as lists, with each batch kept separate.
```
Recommendation just for readability and states what it does normally.

Reply:
thanks, fixed